.gitignore

```diff
@@ -6,22 +6,22 @@
 *.o

```

Comment:
Oh no. You formatted the `.gitignore` as CRLF *after* I made the `.gitignore` LF. Please get rid of these unholy line endings.